README.md

```diff
@@ -19,6 +19,12 @@ Also, I wanted to create something to help my GM and fellow players.
 Once the bot is in your channel, you can use the following to lookup a term and have it delivered directly to your channel:
 * `!lookup <Pathfinder term>`
 
+## Heroku Deployment
+It is possible to deploy the bot on Heroku. **`!lookup` does not currently work when the bot is deployed in this way!**
```

Comment:
I think it does work in Heroku now right?

Reply:
Updated